shared/chat/inbox/row/small-team/index.js

```diff
@@ -91,33 +104,42 @@ class SmallTeam extends React.PureComponent<Props> {
   }
 }
 
-const conversationRowStyle = {
-  ...globalStyles.flexBoxColumn,
-  backgroundColor: isMobile ? globalColors.fastBlank : globalColors.blue5,
-  flexGrow: 1,
-  height: '100%',
-  justifyContent: 'center',
-  paddingLeft: 8,
-  paddingRight: 8,
-}
-
-const conversationRowStyleSelected = {
-  ...conversationRowStyle,
-  backgroundColor: globalColors.blue,
-}
-const rowHeight = isMobile ? 64 : 56
-
-const rowContainerStyle = {
-  ...globalStyles.flexBoxRow,
-  ...desktopStyles.clickable,
-  backgroundColor: isMobile ? globalColors.fastBlank : globalColors.blue5,
-  flexShrink: 0,
-  height: rowHeight,
-}
-
-const rowContainerStyleSelected = {
-  ...rowContainerStyle,
-  backgroundColor: globalColors.blue,
-}
+const styles = styleSheetCreate({
+  container: {flexShrink: 0, height: RowSizes.smallRowHeight},
+  conversationRow: {
+    ...globalStyles.flexBoxColumn,
+    backgroundColor: isMobile ? globalColors.fastBlank : globalColors.blue5,
+    flexGrow: 1,
+    height: '100%',
+    justifyContent: 'center',
+    paddingLeft: 8,
+    paddingRight: 8,
+  },
+  conversationRowSelected: {
+    ...globalStyles.flexBoxColumn,
+    backgroundColor: globalColors.blue,
+    flexGrow: 1,
+    height: '100%',
+    justifyContent: 'center',
+    paddingLeft: 8,
+    paddingRight: 8,
+  },
+  rowContainer: platformStyles({
```

Comment:
Is `platformStyles` what you were telling me about during lunch? Could you possibly explain a little more how it's different from and how it interacts with the platform constants? Is it just to satisfy typing with Flow?

Reply:
Yup, it's the one that Nojima was talking about at lunch the other day. Take a look at it's definition to see how it works: https://github.com/keybase/client/blob/master/shared/styles/shared.js#L48